spdmlib/src/common/mod.rs

```diff
@@ -589,6 +674,15 @@ pub struct SpdmNegotiateInfo {
     pub aead_sel: SpdmAeadAlgo,
     pub req_asym_sel: SpdmReqAsymAlgo,
     pub key_schedule_sel: SpdmKeyScheduleAlgo,
+    pub opaque_data_support: bool,
+    pub opaque_data_fmt0: bool, // ignore when opaque_data_support is false;
+    // this field shall indicate that the format for
+    // all OpaqueData fields in this specification
+    // is defined by the device vendor or other standards body.
+    pub opaque_data_fmt1: bool, // ignore when opaque_data_support is false;
+                                // this field shall indicate that the format for
+                                // all OpaqueData fields in this specification
+                                // is defined by the General opaque data format.
```

Comment:
Please dont use 3 bool.
Please just use 1 field to describe the opaque support.

Reply:
Good point, I created a SpdmOpaqueSupport bitflags variable to do this kind of things. Please check the updated one, Thank you!